Restaurant.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+git<!DOCTYPE html>
```

Comment:
this is not correct html structure. remove git from the document

Reply:
Sure. I will